### PR TITLE
refactor(app): App use textTransform from typography

### PR DESCRIPTION
**Pull request by koji**
# Overview
This is house keeping.
Use  textTransform, textDecoration prop from `TYPOGRAPHY`.
Update the components that are updated within 12 months.

# Changelog
- change to `TYPOGRAPHY.textTransformCapitalize`  from  `TEXT_TRANSFORM_CAPITALIZE`
- change to `TYPOGRAPHY.textTransformUppercase` from `TEXT_TRANSFORM_UPPERCASE`
- change to `TYPOGRAPHY.textDecorationUnderline` from `TEXT_DECORATION_UNDERLINE`

# Review requests
The updated props are the same as the previous
TEXT_TRANSFORM_CAPITALIZE --> TYPOGRAPHY.textTransformCapitalize
TEXT_TRANSFORM_UPPERCASE --> TYPOGRAPHY.textTransformUppercase
TEXT_DECORATION_UNDERLINE --> TYPOGRAPHY.textDecorationUnderline

# Risk assessment
low

### CODE REVIEW

**Review by jerader — APPROVED** [review]
wow thanks for taking this on KojI!

**Comment by koji** [comment]
Changed the target to `edge`.
The timing will after merging `release_6.0.0.` to avoid any unexpected issues

File: app/src/atoms/buttons/index.tsx [hunk 1]
```diff
@@ -52,7 +52,6 @@ export const SecondaryTertiaryButton = styled(NewSecondaryBtn)`
   color: ${COLORS.blue};
   overflow: no-wrap;
   padding: 0.375rem 0.75rem;
-  text-transform: ${TYPOGRAPHY.textTransformNone};
```
Comment (brenthagen) [inline app/src/atoms/buttons/index.tsx]: we need this `textTransformNone` to override the `NewSecondaryBtn` default uppercase
Screen Shot 2022-07-18 at 2 35 29 PM
:

Comment (brenthagen) [reply app/src/atoms/buttons/index.tsx]: and probably the two below as well

Comment (koji) [reply app/src/atoms/buttons/index.tsx]: Thank you for pointing out that.

**Comment by koji** [comment]
revert `none` to buttons because of `BUTTON_VARIANT_STYLE`
https://github.com/Opentrons/opentrons/blob/edge/components/src/primitives/Btn.tsx#L38

**Review by brenthagen — APPROVED** [review]
💯

**Pull request merged**